src/Tests/Aggregations/Bucket/Terms/PartitionTermsAggregationUsageTests.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using FluentAssertions;
+using Nest;
+using Tests.Framework;
+using Tests.Framework.Integration;
+using Tests.Framework.MockData;
+
+namespace Tests.Aggregations.Bucket.Terms
+{
+	[SkipVersion("<5.2.0", "Partitioning term aggregations responses is a new feature in 5.2.0")]
+	public class PartitionTermsAggregationUsageTests : AggregationUsageTestBase
```

Comment:
Maybe `TermsAggregationPartitionUsageTests`? 

For documentation, I think it may be better for this to live in the `TermsAggregationUsageTests` file so that it will appear on https://www.elastic.co/guide/en/elasticsearch/client/net-api/5.x/terms-aggregation-usage.html (The file also needs asciidoc floating headers in it to explain the different examples).